pkg/sfu/buffer/helpers.go

```diff
@@ -278,10 +294,65 @@ func IsH264Keyframe(payload []byte) bool {
 	return false
 }
 
-// IsAV1Keyframe detects if av1 payload is a keyframe
+// -------------------------------------
+
+func IsVP9KeyFrame(payload []byte) bool {
```

Comment:
@cnderrauber Replacing with the simpler version from RFC. Checked that the results match with Jech's implementation.